src/components/WarningBanner/index.tsx

```diff
@@ -0,0 +1,54 @@
+
+import clsx from 'clsx';
+
+import { KUSAMA } from 'utils/constants/relay-chain-names';
+import { LIGHT_DARK_BORDER_CLASSES } from 'utils/constants/styles';
+import { ReactComponent as InformationCircleIcon } from 'assets/img/hero-icons/information-circle.svg';
+
+interface CustomProps {
+  message: string;
+}
+
+type Props = CustomProps & React.ComponentPropsWithRef<'div'>;
+
+const WarningBanner = ({
+  message,
+  className,
+  ...rest
+}: Props): JSX.Element => {
+  return (
+    <div
+      className={clsx(
+        'flex',
+        'items-center',
+        'px-8',
+        'py-3',
+        'space-x-3',
+        'sm:rounded-lg',
+        // TODO: Interlay version is missing
+        // TODO: placeholder color
```

Comment:
The provided color is not from the design system and branding color scheme so I just used a similar color and added a TODO.

Reply:
Why not use the color that Jay provided?

Reply:
@nud3l 
Because I found that color is not from our Kintsugi design system. I used a similar color from Kintsugi color scheme for consistency.

Reply:
Ok, for the future, we'll adjust the color scheme since the design system colors are not very nice to use inside of the app. But that'll start with Jay doing the designs.